kuberos.go

```diff
@@ -3,23 +3,30 @@ package kuberos
 import (
 	"crypto/sha256"
 	"encoding/json"
+	"flag"
 	"fmt"
 	"io/ioutil"
+	"k8s.io/client-go/kubernetes"
```

Comment:
Imports should be sorted such that stdlib imports are separated by a blank line from third party imports.